docs/dask_cudf/source/index.rst

```diff
@@ -6,8 +6,9 @@
 Welcome to dask-cudf's documentation!
 =====================================
 
-Dask-cuDF is an extension library for the `Dask <https://dask.org>`__
-parallel computing framework that provides a `cuDF
+**Dask-cuDF** (pronounced "DASK KOO-dee-eff") is an extension
```

Comment:
Small request: we have a dash in `KOO-dee-eff` but not the other pronunciation guides (`KOO dee-eff`). Should we remove it here, or add it in the other two? I'm fine with either option.